index.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(precompile) {
             throw file.errorWithNode(node, msg);
           }
 
-          this.remove();
+          this.dangerouslyRemove();
```

Comment:
Do you think it makes sense to make this backwards compatible? Because if only the `babel-plugin-htmlbars-inline-precompile` package is updated, but the pugin is used with an older version of babel (`dangerouslyRemove` has been introduced in [v5.5.0](https://github.com/babel/babel/commit/98eb30c4828e5210f767180ae1aa19db21334a82)), an error would be thrown, since only the *old* `remove` would be available ...

What do you think?